arch/arm64/boot/dts/qcom/sm8150-kumano-griffin-display.dtsi

```diff
@@ -49,6 +49,10 @@
 			qcom,default-topology-index = <0>;
 			qcom,mdss-dsi-panel-phy-timings = [00 21 08 08 25 22 09 08 06 02 04 00 1B 18];
 		};
+
+		/* For Google Maps crazy resolution switching issue... */
+		/delete-node/ timing@0;
```

Comment:
Care to elaborate?

Reply:
There's a crazy resolution switch back and forth being called by Google Maps app.
Some users rely on this app for literally everything, so it's critical for them to have it working.

By the way, when this happens, the app is totally unusable.
There is currently no way to fix this in userspace: Google is aware of the issues that their app has, infact they have added a switch blacklist that's currently being used by Coral, but that's only for refresh rate switches and not for resolutions.

I hope that the right way of fixing this will come in the next Maps app updates, or in the next
updates in Android's blacklist, so that I can revert this specific commit.
In any case, the lower resolution is not ever used by the Griffin device since the permanent switch to the 4k one so - despite this being a mere hack - noone is gonna miss it at all.